Sources/CoffeeBot/CoffeeManager/CoffeeManager.swift

```diff
@@ -0,0 +1,94 @@
+//
+//  CoffeeManager.swift
+//  SwiftBot
+//
+
+import Foundation
+import BotsKit
+import LoggerAPI
+
+internal protocol CoffeeManagerProtocol {
+    func process(activity:Activity)
+}
+
+class CoffeeManager {
+    
+    //MARK: Properties
+    fileprivate var session : Session? = nil
+    fileprivate let textProcessingService : TextProcessingService
+    fileprivate let appointingService : AppointingService
+    fileprivate let sessionService : SessionService
+    weak var botDelegate : CoffeeBot?
+    
+    //MARK: Lifecycle
+    init(textProcessingService:TextProcessingService, appointingService:AppointingService, sessionService:SessionService) {
```

Comment:
That is a really good point, I will think of this. I only have one concern which is:
If I receive something like ```init(services:[CoffeeBotServiceProtocol])``` since all services have different functionality and method signatures I am not sure if I can unify them.